Yoast/javascript
# Create TwitterDescription component

## Summary
* Adds a TwitterDescription component

## Relevant technical choices:

* We (@hwinne and I) have added a `type` property (`summary` versus `summary-large-image`), which is used to determine the maximum height of the description, and the width of the description.
* We no longer care about a maximum number of characters in the description (the issue originally said the max should be 200), because truncation is determined based on the number of lines, rather than on the number of characters.
* We did not yet implement ellipsis (`...`) because this has moved to another issue (#157).

## Test instructions

This PR can be tested by following these steps:

* Check out this branch.
* From the `javascript/apps/components` folder in the terminal, run `yarn` and `yarn start`.
* `yarn test` should not result in any errors.
* Start the stand-alone by navigating to `http://localhost:3333/` in the browser.
* Go to the `TwitterPreview` page by clicking on the button in the top right.
* Check if the descriptions look alright.
    * If the card type is `Summary with large image`, the long description should maximally span two lines, and have a width of `476px`.
    * If the card type is `Summary`, the long description should maximally span three lines, and have a width of `357px`.

## UI changes

## Quality assurance


Fixes #49
